central/declarativeconfig/updater/role_updater.go

```diff
@@ -0,0 +1,33 @@
+package updater
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+	roleDataStore "github.com/stackrox/rox/central/role/datastore"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+)
+
+// RoleUpdater handles updates of storage.Role within declarative config reconciliation routine.
+type RoleUpdater struct {
```

Comment:
no reason to export those, right?
```suggestion
type roleUpdater struct {
```